contracts/modules/RecoveryManager.sol

```diff
@@ -145,9 +145,11 @@ contract RecoveryManager is BaseModule, RelayerModule {
      */
     function cancelRecovery(BaseWallet _wallet) external onlyExecute onlyWhenRecovery(_wallet) {
         RecoveryConfig storage config = recoveryConfigs[address(_wallet)];
-        emit RecoveryCanceled(address(_wallet), config.recovery);
-        guardianStorage.setLock(_wallet, 0);
+        address recoveryOwner = config.recovery;
```

Comment:
Since `config` is only used to define `recoveryOwner` it is probably more efficient to do
```
address recoveryOwner = recoveryConfigs[address(_wallet)].recovery;
```